File: app/src/main/java/in/ceeq/lyte/databinding/ContactListViewModel.java
package in.ceeq.lyte.databinding;


import android.content.Context;
import android.view.View;

import in.ceeq.lyte.ContactEditActivity;

public class ContactListViewModel extends BaseContactViewModel {

    public ContactListViewModel(Context context) {
        super(context);
    }

    public void onAddContactClick(View view) {
        ContactEditActivity.start(view.getContext());
    }
}

File: app/src/main/java/in/ceeq/lyte/databinding/ContactListViewModel.kt
package `in`.ceeq.lyte.databinding


import `in`.ceeq.lyte.user.UserEditActivity
import android.content.Context
import android.view.View

class ContactListViewModel(context: Context) : BaseContactViewModel(context) {

    fun onAddContactClick(view: View) {
        UserEditActivity.start(view.context)
    }
}
